docs/user/guides/canonical-urls.rst

```diff
@@ -23,7 +28,27 @@ MkDocs
 For :doc:`MkDocs </intro/getting-started-with-mkdocs>` we do not define your canonical domain automatically,
 but you can use the site_url_ setting to set a similar value.
 
+In your ``mkdocs.yml``, define the following:
+
+.. code-block:: yaml
+
+   # Canonical URL, adjust as need with respect to your slug, language,
+   # default branch and if you use a custom domain.
+   site_url: https://<slug>.readthedocs.io/en/stable/
+
+Note that this will define the same canonical URL for all your branches and versions.
+According to MkDocs, defining site_url_ only affects the canonical URL of a website.
```

Comment:
I actually removed my elaboration because it became too lengthy.

```suggestion
According to MkDocs, defining site_url_ will only define the canonical URL of a website and does not affect the base URL of for instance links or included CSS or JS files.
```

Reply:
Edited yours to remove the `for instance` and clarify it a bit more 👍